src/status_im/ethereum/json_rpc.cljs

```diff
@@ -6,6 +6,9 @@
             [status-im.utils.utils :as utils]
             [taoensso.timbre :as log]))
 
+;; TODO (19/12/22 yqrashawn) this namespace has been moved to the status-im2.common.json-rpc.events
```

Comment:
can you make a corresponding issue on github to delete this code? 
And if there is some documentation that we have for actions that need to do after the release then can you add this issue to that release doc. 
cc @cammellos

Reply:
we'll remove the entire status-im package i guess

Reply:
Yes, these comments are for devs that looking for functions to use in old namespace. The deletion will happen in one go